frontend/src/metabase/browse/components/BrowseApp.tsx

```diff
@@ -40,6 +42,7 @@ export const BrowseApp = ({
       filter_items_in_personal_collection: "exclude",
     },
   });
+  const collectionsResult = useCollectionListQuery();
```

Comment:
Collections are now retrieved so that we can use the correct icon for Metabase
analytics, which is the "audit" icon